schema/validator.go

```diff
@@ -219,7 +246,7 @@ func checkPlatform(OS string, Architecture string) {
 					return
 				}
 			}
-			fmt.Printf("warning: combination of %q and %q is invalid.\n", OS, Architecture)
+			fmt.Printf("warning: combination of OS %q and architecture %q is invalid.\n", OS, Architecture)
```

Comment:
nit: maybe uncapitalize `OS` to match the field name?

Reply:
Fixed